src/redux/actions/index.js

```diff
@@ -63,6 +107,13 @@ export function addNewTerm(newTerm) {
 
 export function synchronizeStorageToRedux() {
   return async dispatch => {
+    const storedLearntPhrases = await getDataFromStorage(LEARNT_PHRASES_KEY);
+
+    if (!storedLearntPhrases) {
+      return Promise.resolve();
+    }
+
+    dispatch(setLearntPhrases(storedLearntPhrases));
     const storedNewTerms = await getData(NEW_TERMS_KEY);
     if (!storedNewTerms) {
```

Comment:
So here the imported function for storing data in storage will be used.
Should be like this ` const storedLearntPhrases = await getDataFromStorage(LEARNT_PHRASES_KEY);` and `const storedNewTerms = await getDataFromStorage(NEW_TERMS_KEY);`